src/ontology/config/icd10cm_exclusions.tsv

```diff
@@ -0,0 +1,725 @@
+term_id	term_label	exclusion_reason	exclude_children

```

Comment:
There are actually some problems with this file. What I have here had to be massaged from our previous work that we had done and put in Google Sheets. But there were actually a number of problems with those results, namely that the input list of intensional terms had some problems. I'll bring up what those were in the next QC meeting.

For now, I created a GitHub issue about the issues:
https://github.com/monarch-initiative/mondo-analysis/issues/38